### ISSUE TITLE
[Feature Request] Ability to filter subreddits

### ISSUE THREAD

**Issue by kittles** [michael-lazar/rtv#466]
would be nice to be able to configure ignored subreddits

**Comment by fullstopslash** [comment]
Can't you just unsubscribe from a sub to ignore it? Or am I missing something here?

**Comment by kittles** [comment]
ah sorry, its nice to have the ability to blacklist when browsing /r/all for example.

**Comment by kbknapp** [comment]
Yes, this would be awesome. There are times when I only want to temporarily filter out a specific subreddit or two while browsing my front page, but I don't want to actually unsubscribe.

**Comment by mredaelli** [comment]
I'd also like this, but for another reason.
I am subscribed to, among other things, stuff like /r/blackmagicfuckery or /r/CatsAreAssholes, which are image-heavy, and don't really make much sense browsing in rtv - I prefer seeing the preview comfortably from my phone.
But I would prefer rtv for "text stuff"
For

**Comment by mredaelli** [comment]
At a quick glance, a hacky solution (simply storing a list of subreddits to exclude in the config file) doesn't seem too complicated. Would you accept a PR?

**Comment by mattydebie** [comment]
I would be interested in this as well!
